src/components/sequencing/form/SelectedCommand.svelte

```diff
@@ -191,21 +193,29 @@
   // {'unsigned_arg', 'enum_arg', 'var_string_arg', 'float_arg'}
 </script>
 
-<div class="select-command-detail" id={ID_COMMAND_DETAIL_PANE}>
-  {#if !!commandNode}
-    <div>Selected Command</div>
-    {#if !!commandDef}
-      {#if !!timeTagNode}
-        <div>Time Tag: {timeTagNode.text.trim()}</div>
-      {/if}
-      <div>
-        <details>
-          <summary>{commandDef.stem}</summary>
-          {commandDef.description}
-        </details>
-      </div>
-      <hr />
-      <div class="select-command-argument-detail">
+<Panel overflowYBody="hidden" padBody={false}>
+  <svelte:fragment slot="header">
+    <SectionTitle>Selected Command</SectionTitle>
+  </svelte:fragment>
+
+  <svelte:fragment slot="body">
+    {#if !!commandNode}
+      <div class="header"></div>
+
+      {#if !!commandDef}
+        {#if !!timeTagNode}
+          <fieldset>
+            <div>Time Tag: {timeTagNode.text.trim()}</div>
+          </fieldset>
+        {/if}
+
+        <fieldset>
```

Comment:
Could you use the `Collapse` component here? I can also take a crack at styling this if that'd be helpful.

Reply:
Done